src/components/login/login.js

```diff
@@ -25,10 +25,26 @@ class Login extends React.Component {
   constructor(props) {
     super(props);
 
+    const { autologinUrl } = getAutoLogInData();
+
+    let loginNetwork = Object.entries(networks).find((network) => {
+      const { nodes } = network.slice(-1).shift();
+      return Array.isArray(nodes) ? nodes.includes(autologinUrl) : false;
+    });
+
+    let address = '';
+
+    if (loginNetwork) {
+      loginNetwork = loginNetwork.slice(-1).shift();
+    } else if (!loginNetwork) {
+      loginNetwork = autologinUrl ? networks.customNode : networks.default;
+      address = autologinUrl;
+    }
+
     this.state = {
       passphrase: '',
-      address: '',
-      network: networks.default.code,
+      address,
+      network: loginNetwork.code, // networks.default.code,
```

Comment:
What is the comment trying to say here?

```suggestion
      network: loginNetwork.code,
```